packages/client/src/shared/ui/BackgroundAudioSwitcher/ui/BackgroundAudioSwitcher.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+import BackgroundMusic from '@/shared/assets/sounds/backgroundMusic.mp3';
+import AudioOff from '@/shared/assets/images/audioOff.png';
+import AudioOn from '@/shared/assets/images/audioOn.png';
+import classNames from 'classnames';
+import css from './BackgroundAudioSwitcher.module.scss';
+
+export const BackgroundAudioSwitcher = () => {
+  const [audio, setAudio] = useState(undefined as unknown as HTMLAudioElement);
```

Comment:
```suggestion
  const [audio, setAudio] = useState<HTMLAudioElement>();
```
Тип лучше указать таким образом